setup.py

```diff
@@ -2,7 +2,7 @@
 
 setup(
     name='django-markwhat',
-    version=".".join(map(str, __import__('django_markwhat').__version__)),
```

Comment:
I hardcoded the version number here because this line was giving me an error when I tried to run `tox` on my checkout.